docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -0,0 +1,121 @@
+[//]: # (title: Compiler options)
+
+Using a build script, you can specify additional compilation options. 
```

Comment:
This is the new sentence. Everything ahead is old except the "What's next" section.

Reply:
Maybe an example of options? 
If the audience doesn't know about compilation options, should we provide some examples to explain what will be in the article?